### PR TITLE
Updated the build-script and added build_info.json and README.md for …

### PR DESCRIPTION
**Pull request by Pratikt2312**
…opentelemetry-js-contrib

## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@nikhil-kalbande please review

**Comment by nikhil-kalbande** [comment]
@Pratikt2312 why we are not using latest tag/version `auto-instrumentations-node-v0.36.0` and changed to use version `instrumentation-user-interaction-*`
Also please explain idea behind adding Readme file here.

**Comment by kandarpamalipeddi** [comment]
> @Pratikt2312 why we are not using latest tag/version `auto-instrumentations-node-v0.36.0` and changed to use version `instrumentation-user-interaction-*` Also please explain idea behind adding Readme file here.

@nikhil-kalbande 

1. This project/repo got different tag names (they are not following a single style, they got different names depending on the supported features/versions. Hence we need to conclude on `regex` which we need to use here.
2. While testing this package, we found that, it is using MongoDB, hence we need to start the MongoDB container and export couple of environment variables to perform successful connection.
Currently, we are not supporting, docker inside docker in travis check, I asked @Pratikt2312  to mention all the steps in ReadMe, skip testing and set Travis flag to false.

File: o/opentelemetry-js-contrib/build_info.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "package_name" : "opentelemetry-js-contrib",
+    "github_url":"https://github.com/open-telemetry/opentelemetry-js-contrib",
+    "version": "auto-instrumentations-node-v0.36.0",
+    "package_dir": "o/opentelemetry-js-contrib",
+    "docker_build": false,
+    "build_script" : "opentelemetry-js-contrib_ubi_8.5.sh",
+    
+    "auto-instrumentations-node-v0.*.*" : {
```
Comment (kandarpamalipeddi) [inline o/opentelemetry-js-contrib/build_info.json:9]: Discuss with @seth-priya & @adilhusain-s to confirm on which versions we are going to support.

**Review by kandarpamalipeddi — APPROVED** [review]

File: o/opentelemetry-js-contrib/build_info.json (hunk 1 above)
Comment (kandarpamalipeddi) [reply o/opentelemetry-js-contrib/build_info.json:9]: @Pratikt2312 
Please discuss with team and confirm on version, rest is looks good to me.

Comment (Pratikt2312) [reply o/opentelemetry-js-contrib/build_info.json:9]: ok sure.

**Comment by seth-priya** [comment]
@nikhil-kalbande @Pratikt2312  where are we with this one?

**Comment by nikhil-kalbande** [comment]
> @nikhil-kalbande @Pratikt2312 where are we with this one?

@seth-priya - Last as Kandarpa mentioned there is confusion on version to support as there are 2 different patterns in tags/version like `auto-instrumentations-node-v*` and `instrumentation-user-interaction-* `.  @Pratikt2312 do we have conclusion on this?
@seth-priya please guide if you have insight on this.

**Comment by Pratikt2312** [comment]
> > @nikhil-kalbande @Pratikt2312 where are we with this one?
> 
> @seth-priya - Last as Kandarpa mentioned there is confusion on version to support as there are 2 different patterns in tags/version like `auto-instrumentations-node-v*` and `instrumentation-user-interaction-* `. @Pratikt2312 do we have conclusion on this? @seth-priya please guide if you have insight on this.

@seth-priya @nikhil-kalbande - As mentioned by Kandarpa this repo has different tags/versions, so as per previous build script first i checked it for auto-instrumentations-node-v* and then also checked for instrumentation-user-interaction-*  that was latest when i worked on this, so for both versions its working fine.Once will discuss with @adilhusain-s regarding this version confirmation.

**Comment by kandarpamalipeddi** [comment]
> > > @nikhil-kalbande @Pratikt2312 where are we with this one?
> > 
> > 
> > @seth-priya - Last as Kandarpa mentioned there is confusion on version to support as there are 2 different patterns in tags/version like `auto-instrumentations-node-v*` and `instrumentation-user-interaction-* `. @Pratikt2312 do we have conclusion on this? @seth-priya please guide if you have insight on this.
> 
> @seth-priya @nikhil-kalbande - As mentioned by Kandarpa this repo has different tags/versions, so as per previous build script first i checked it for auto-instrumentations-node-v* and then also checked for instrumentation-user-interaction-* that was latest when i worked on this, so for both versions its working fine.Once will discuss with @adilhusain-s regarding this version confirmation.


If it is working fine with the both the versions, can you add an entry for `instrumentation-user-interaction-*` as well ? sample code as below :
```
"auto-instrumentations-node-v0.*.*" : {
	"build_script" : "opentelemetry-js-contrib_ubi_8.5.sh",
	"args": {
	      "NODE_VERSION":"16"
        }
    } 
"instrumentation-user-interaction-v0.*.*" : {
	"build_script" : "opentelemetry-js-contrib_ubi_8.5.sh",
	"args": {
	      "NODE_VERSION":"16"
        }
    } 
```

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Update the build-info file for both the version tags.

**Comment by nikhil-kalbande** [comment]
> Update the build-info file for both the version tags.

yes @kandarpamalipeddi , we discussed and decided on same. :-)

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**